### PR TITLE
build(travis): fix travis environment to build es2015 distro

### PR DESCRIPTION
**Pull request by alexeagle**
Too inconvenient to test master-upstream-only Travis features :(
I misread the docs, you can't set multiple variables on a matrix line, only on env lines:
https://docs.travis-ci.com/user/environment-variables/#Defining-Multiple-Variables-per-Item

Just do this in env.sh

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -28,12 +28,12 @@ env:
...
-   - CI_MODE=saucelabs_required
-   - CI_MODE=browserstack_required
-   - CI_MODE=saucelabs_optional
-   - CI_MODE=browserstack_optional
+    - CI_MODE=e2e
+    - CI_MODE=js
+    - CI_MODE=saucelabs_required
+    - CI_MODE=browserstack_required
+    - CI_MODE=saucelabs_optional
+    - CI_MODE=browserstack_optional
```
Comment (vicb) [inline .travis.yml]: no diff ? revert ?

File: scripts/ci-lite/env.sh [hunk 2]
```diff
@@ -58,6 +58,11 @@ if [[ ${TRAVIS} ]]; then
   # more info: https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Node.js-v4-(or-io.js-v3)-compiler-requirements
   export CXX=g++-4.8
 
+  # Build ES2015 distro only in the e2e job, which publishes snapshot builds
+  if [ "${CI_MODE}" = "e2e" ]; then
+    export EXPERIMENTAL_ES2015_DISTRO=1
```
Comment (vicb) [inline scripts/ci-lite/env.sh]: should we have an `export PUBLISH_SNAPSHOT=1` instead of a comment here ?

File: .travis.yml (hunk 1 above)
Comment (alexeagle) [inline .travis.yml]: seems like some bad merge happened, my last PR added a variable here: https://github.com/angular/angular/pull/13471/files

File: scripts/ci-lite/env.sh (hunk 2 above)
Comment (alexeagle) [inline scripts/ci-lite/env.sh]: we always publish a snapshot for successful travis runs on master. If we find an es2015 distro, we publish that too.

This causes us to build the es2015 distro.

Comment (alexeagle) [inline scripts/ci-lite/env.sh]: updated the comment, PTAL (this file doesn't control whether snapshots are published)

File: .travis.yml (hunk 1 above)
Comment (alexeagle) [inline .travis.yml]: reverted.

**Comment by alexeagle** [comment]
PTAL

**Comment by IgorMinar** [comment]
can you or the caretaker change the commit message to be `ci: ` rather than `build(travis)`? this change has nothing to do with build.

**Comment by IgorMinar** [comment]
otherwise lgtm

**Comment by alexeagle** [comment]
done

**Pull request merged**